manual/parts/objects.tex

```diff
@@ -44,8 +44,8 @@ \chapter{\label{ch:objects}Kernel Services and Objects}
     kernel services along with their book-keeping information.
 \end{description}
 
-This chapter gives an overview of these services, describes how kernel
-objects are accessed by userspace applications, and describes how new
+This chapter gives an overview of these services and describes how kernel
```

Comment:
leave the comma here?

Reply:
I think the change is correct, there should not be a comma here in English (actually in German neither ;-))